app-react/src/jsMain/kotlin/com/ustadmobile/mui/components/SocialWarningListItem.kt

```diff
@@ -0,0 +1,80 @@
+package com.ustadmobile.mui.components
+
+import com.ustadmobile.core.MR
+import com.ustadmobile.core.hooks.useStringProvider
+import js.objects.jso
+import mui.material.Button
+import mui.material.ButtonVariant
+import mui.material.ListItem
+import mui.material.ListItemIcon
+import mui.material.ListItemText
+import mui.material.Stack
+import mui.material.StackDirection
+import mui.system.responsive
+import react.FC
+import react.Props
+import react.ReactNode
+import react.create
+import react.dom.html.ReactHTML.div
+import web.cssom.px
+import web.window.WindowTarget
+import web.window.window
+import mui.icons.material.Warning as WarningIcon
+
+private const val SOCIAL_WARNING_WEB_URL = "https://beinternetawesome.withgoogle.com"
```

Comment:
As per previous comment https://github.com/UstadMobile/UstadMobile/pull/980#discussion_r1856554102
"

This constant should be declared on a related use case (e.g. ShowSocialWarningUseCase), not in this viewmodel. It should be declared once, such that if this link needs to change, there should be one place to go and change it.

This code as-is does implement single point of truth as per coding style - the same constant is being declared in multiple places.
"